### PR TITLE
UI tweaks for org management

### PR DESCRIPTION
**Pull request by orangejenny**
Followup for http://manage.dimagi.com/default.asp?220811
- added individual download button
- matching colors & icons

@sravfeyn / @emord

### CODE REVIEW

File: corehq/apps/locations/templates/locations/manage/locations.html [hunk 1]
```diff
@@ -188,14 +188,23 @@
 <div class="row">
     <div class="col-sm-12">
         <div class="btn-toolbar">
-            <a class="btn btn-primary"
+            <a class="btn btn-info"
+               {% if not has_location_types or not can_edit_any_location %}
```
Comment (emord) [inline corehq/apps/locations/templates/locations/manage/locations.html]: I'm not familiar with the location permissions, but is it possible for someone to be able to view, but not edit locations?

Comment (orangejenny) [inline corehq/apps/locations/templates/locations/manage/locations.html]: Given that the existing upload/download link is limited to users with edit permissions, this PR isn't making things any worse.

@esoergel should it be possible for someone who can't edit locations to still be able to download them?

**Comment by emord** [comment]
build fail looked unrelated. kicked build

File: corehq/apps/locations/templates/locations/manage/locations.html (hunk 1 above)
Comment (esoergel) [inline corehq/apps/locations/templates/locations/manage/locations.html]: I agree that it's probably fine to leave as-is.

Unfortunately we don't really have a cohesive system for location-based permissions, which is something I've talked with product about.  In general though, if access is restricted by locations, then only domain admins can make changes to locations.  I don't think we have a policy on viewing locations.

**Comment by czue** [comment]
thanks for grabbing this!

**Comment by biyeun** [comment]
:tada:

**Pull request merged**